### ISSUE TITLE
Lost prop

### ISSUE THREAD

**Issue by djvonavi** [mengxiong10/vue2-datepicker#294]
Guys, hello. Your property has been lost `firstDayOfWeek`. And for this you can not customize the days of the week. In the main file `/src/index.vue`. This component send default number 7 to calendar.vue. Can U fix that trouble? Thnks!

**Comment by mengxiong10** [comment]
It is passed through $attrs.
Does you set but not workinng?

**Comment by djvonavi** [comment]
@mengxiong10 Strange, why then he ignores my properties?
```
<DatePicker
    v-model="date1"
    placeholder="Дата с"
    lang="ru"
    format="DD.MM.YYYY"
   :first-day-of-week="1" />
```
<img width="318" alt="Снимок экрана 2019-04-16 в 16 55 48" src="https://user-images.githubusercontent.com/11898500/56215614-9ab61400-6068-11e9-9b14-79fc2c882325.png">
<img width="488" alt="Снимок экрана 2019-04-12 в 17 24 38" src="https://user-images.githubusercontent.com/11898500/56215682-b3bec500-6068-11e9-815b-264da93cad25.png">

**Comment by djvonavi** [comment]
Sorry, this issue must be closed) `"vue": "2.3.4"` my build very old(
